wagtail/documents/templates/wagtaildocs/documents/edit.html

```diff
@@ -1,63 +1,33 @@
-{% extends "wagtailadmin/base.html" %}
-{% load i18n %}
-{% load wagtailimages_tags wagtailadmin_tags %}
-{% block titletag %}{% blocktrans trimmed with title=document.title %}Editing {{ title }}{% endblocktrans %}{% endblock %}
+{% extends "wagtailadmin/generic/edit.html" %}
+{% load i18n wagtailadmin_tags %}
 
-{% block extra_js %}
-    {{ block.super }}
-
-    {{ form.media.js }}
-{% endblock %}
-
-{% block extra_css %}
-    {{ block.super }}
-    {{ form.media.css }}
-{% endblock %}
-
-{% block content %}
-    {% trans "Editing" as editing_str %}
-    {% include "wagtailadmin/shared/header.html" with title=editing_str subtitle=document.title icon="doc-full-inverse" %}
-    {% include "wagtailadmin/shared/non_field_errors.html" %}
-
-    <div class="row row-flush nice-padding">
```

Comment:
I believe we're moving away from these legacy `row`/`col` classes to the more modern `grid` display, so I refactored them as well.